bulk-export/src/test/java/no/sikt/nva/data/report/api/export/GenerateKeyBatchesHandlerTest.java

```diff
@@ -174,7 +175,7 @@ private KeyBatchRequestEvent saveContainedEvent(PutEventsRequest putEventsReques
             PutEventsRequestEntry eventEntry = putEventsRequest.entries()
                                                    .stream()
                                                    .collect(SingletonCollector.collect());
-            return attempt(eventEntry::detail).map(
+            return Try.attempt(eventEntry::detail).map(
```

Comment:
Why try?

Reply:
Some say one just must —
Ambition is critical!
Here, though, JetBrains helped.